This is workaround until fix for 24750280 is delivered to build machine.

Not suitable for upstream.

--- ImageMagick-6.9.5/coders/tiff.c.old	2016-09-28 07:57:25.320767505 -0700
+++ ImageMagick-6.9.5/coders/tiff.c	2016-09-28 07:57:27.957254880 -0700
@@ -84,6 +84,19 @@
 #include "magick/thread_.h"
 #include "magick/token.h"
 #include "magick/utility.h"
+
+#if !defined(__sparcv9)
+# if !defined(__amd64)
+#  include "tiffconf.h"
+#  undef TIFF_INT64_T
+#  define TIFF_INT64_T signed long long
+#  undef TIFF_UINT64_T
+#  define TIFF_UINT64_T unsigned long long
+#  undef TIFF_SSIZE_T
+#  define TIFF_SSIZE_T signed int
+# endif
+#endif
+
 #if defined(MAGICKCORE_TIFF_DELEGATE)
 # if defined(MAGICKCORE_HAVE_TIFFCONF_H)
 #  include "tiffconf.h"
